homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -38,6 +38,7 @@
 SERVICE_MOVE_REMOTE_CONTROL_STEP = 'xiaomi_remote_control_move_step'
 SERVICE_START_REMOTE_CONTROL = 'xiaomi_remote_control_start'
 SERVICE_STOP_REMOTE_CONTROL = 'xiaomi_remote_control_stop'
+SERVICE_START_CLEAN_ZONE = 'xiaomi_clean_zone_start'
```

Comment:
I think `xiaomi_clean_zone` sounds better, as there is no `xiaomi_clean_zone_stop` or anything like that.

Reply:
fixed